lib/PieChart.js

```diff
@@ -26,7 +26,10 @@ const iface = {
     // TODO PieChart should have only one dataset
     data: PropTypes.shape({
       datasets: PropTypes.arrayOf(PropTypes.shape({
-        yValues: PropTypes.arrayOf(PropTypes.number),
+        values: PropTypes.arrayOf(PropTypes.shape({
+          y: PropTypes.number.isRequired,
```

Comment:
why not `value` instead of `y` for this chart? Please also change that for Radar chart